specification/protocol/generate_rest.yaml

```diff
@@ -0,0 +1,89 @@
+openapi: 3.1.0
+info:
+  title: Open Inference API for text generation
+  description: Open Inference API for text generation
+  version: 1.0.0
+components:
+  schemas:
+    GenerateRequest:
+      type: object
+      required: 
+        - text_input
+      properties:
+        text_input:
+          type: string
+        parameters:
+          $ref: '#/components/schemas/GenerateParameters'
+    GenerateParameters:
+      type: object
+      properties:
+        parameter:
+          description: Parameters are framework specific. An optional object containing zero or more parameters for this generate request expressed as key/value pairs 
+    GenerateResponse:
+      type: object
+      required:
+        - text_output
+        - model_name
+        - model_version
+      properties:
+        text_output:
+          type: string
+        model_name:
+          type: string
+        model_version:
+          type: string
+    GenerateStreamResponse:
+      type: object
+      required:
+        - text_output
+        - model_name
+        - model_version
+        - done
+      properties:
+        text_output:
```

Comment:
This is concatenated text output, we might still want to see the token generated for each iteration.

Reply:
In the Nvidia implementation, each response in returning cumulative set of tokens.
```
1st json
{
text_output: "Here is"
}
.
.
.
..
subsequent json response
{
text_output: "Here is the output for the prompt"
}
```

Should we add additional property to display token generated in current response set?